calico_containers/calicoctl.py

```diff
@@ -437,6 +470,23 @@ def normalize_version(version):
     """
     return [int(x) for x in re.sub(r'(\.0+)*$','', version).split(".")]
 
+
+def get_first_ip():
+    """
+    Searches for the first IP address (that isn't the docker0 veth).
+    Since already has a dependency on 'ip' binary, we'll use that to
+    find the address.
+    """
+    ip = sh.Command._create("ip")
```

Comment:
Wouldn't that attempt to connect to Google's DNS server? What if the container / host doesn't have a connection to the WAN? Wouldn't this fail?